src/v2i-hub/RSUHealthMonitorPlugin/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "RSUHealthMonitor",
+    "description": "Monitor RSU health status",
+    "version": "@PROJECT_VERSION@",
+    "exeLocation": "/bin/RSUHealthMonitorPlugin",
+    "coreIpAddr":"127.0.0.1",
+	"corePort":24601,
+    "messageTypes": [],
+    "configuration": [
+        {
+            "key": "LogLevel",
+            "default": "INFO",
+            "description": "The log level for this plugin"
+        },
+        {
+            "key":"Interval",
+            "default":"1",
+            "description": "Sending RSU SNMP GET request at every configured interval. Default every 1 second. Unit of measure: second."
+        },
+		{
+            "key":"RSUIp",
+            "default":"192.168.XX.XX",
+            "description":"An IP address of the RSU the V2X hub is connected to."
+        },
+		{
+            "key":"SNMPPort",
+            "default":"161",
+            "description":"The SNMP port for sending message or command."
+        },
+		{
+            "key":"AuthPassPhrase",
+            "default":"dummy",
+            "description":"SNMP v3 authentication passphrase"
+        },
+		{
+            "key":"SecurityUser",
+            "default":"authOnlyUser",
+            "description":"SNMP Security Name"
+        },
+		{
+            "key":"RSUOIDConfigMap",
+            "default":"{\"RSUOIDConfig\":[{\"RsuField\":\"rsuGpsOutputString\",\"OID\":\"iso.0.15628.4.1.8.5.0\",\"Required\":false},{\"RsuField\":\"rsuID\",\"OID\":\"iso.0.15628.4.1.17.4.0\",\"Required\":false},{\"RsuField\":\"rsuMibVersion\",\"OID\":\"iso.0.15628.4.1.17.1.0\",\"Required\":false},{\"RsuField\":\"rsuFirmwareVersion\",\"OID\":\"iso.0.15628.4.1.17.2.0 \",\"Required\":false},{\"RsuField\":\"rsuManufacturer\",\"OID\":\"iso.0.15628.4.1.17.5.0\",\"Required\":false},{\"RsuField\":\"rsuIFMIndex\",\"OID\":\"iso.3.6.1.2.1.1.7.0\",\"Required\":false},{\"RsuField\":\"rsuIFMPsid\",\"OID\":\"iso.3.6.1.2.1.1.7.0\",\"Required\":false},{\"RsuField\":\"rsuIFMDsrcMsgId\",\"OID\":\"iso.3.6.1.2.1.1.7.0\",\"Required\":false},{\"RsuField\":\"rsuIFMTxMode\",\"OID\":\"iso.3.6.1.2.1.1.7.0\",\"Required\":false},{\"RsuField\":\"rsuIFMTxChannel\",\"OID\":\"iso.3.6.1.2.1.1.7.0\",\"Required\":false},{\"RsuField\":\"rsuIFMEnable\",\"OID\":\"iso.3.6.1.2.1.1.7.0\",\"Required\":false},{\"RsuField\":\"rsuIFMStatus\",\"OID\":\"iso.3.6.1.2.1.1.7.0\",\"Required\":false},{\"RsuField\":\"rsuMode\",\"OID\":\"iso.0.15628.4.1.99.0\",\"Required\":false}]}",
```

Comment:
This does not make sense to me. Aren't these OIDs standardized. Maybe it would be better to specify a document version here and then have different headers that contain the actual OIDs.